pages/linux/fixfiles.md

```diff
@@ -0,0 +1,36 @@
+# fixfiles
+
+> Fix file SELinux security contexts.
+> More information: <https://manned.org/fixfiles>.
+
+- If specified with onboot, this fixfiles will record the current date in the `/.autorelabel` file, so that it can be used later to speed up labeling. If used with restore, the restore will only affect files that were modified today:
+
+`fixfiles -B`
+
+- Force reset of context to match `file_context` for customizable files:
+
+`fixfiles -F`
+
+- Clear `/tmp` directory with out prompt for removal:
+
+`fixfiles -f`
+
+- Use the rpm database to discover all files within the specified packages and restore the file contexts:
+
+`fixfiles -R rpmpackagename[,rpmpackagename...]`
```

Comment:
```suggestion
- Use the [R]pm database to discover all files within specific packages and restore the file contexts:

`fixfiles -R {{rpm_package_name1,rpm_package_name2 ...}}`
```

Almost the same issue as above.
Optional fix: `specific` may sound better.
Mandatory fix: [ellipsis must be used for several items](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases).